src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    static final int NUMBER_OF_FIGURES = 5;
+    static final String DEFAULT_COLOR = Color.WHITE.name();
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        String randomColor = colorSupplier.getRandomColor();
+        int randomFigure = random.nextInt(NUMBER_OF_FIGURES);
+
+        switch (randomFigure) {
+            case 0:
+                return new Circle();
+            case 1:
+                return new Rectangle();
+            case 2:
+                return new Square();
+            case 3:
+                return new RightTriangle();
+            default:
+                return new IsoscelesTrapezoid();
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(10, DEFAULT_COLOR);
+    }
+
```

Comment:
redundant blank line
```suggestion
```

Reply:
not fixed

Reply:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/156338484/05ef5b73-1e3c-4d45-bf6f-b71a881716d0)